layouts/partials/json-schema/resolve-additional-types.html

```diff
@@ -91,6 +91,20 @@
     {{ end }}
 {{ end }}
 
+{{ if and $this_object.oneOf (reflect.IsSlice $this_object.oneOf) }}
```

Comment:
A comment would be nice:

```suggestion
{{/*
  Handle object schemas using the `oneOf` keyword
  (https://json-schema.org/understanding-json-schema/reference/combining.html#oneof)
*/}}
{{ if and $this_object.oneOf (reflect.IsSlice $this_object.oneOf) }}
```